tests/test_project.py

```diff
@@ -627,6 +627,32 @@ def test_project_status_homogeneous_schema(self):
                 with redirect_stderr(StringIO()):
                     project.print_status(parameters=parameters, detailed=True)
 
+    def test_concurrent_project_status_homogeneous_schema(self):
```

Comment:
Technically this is testing serial execution
```suggestion
    def test_serial_project_status_homogeneous_schema(self):
```